build-system/git.js

```diff
@@ -60,17 +60,23 @@ exports.gitDiffStatMaster = function() {
 
 /**
  * Returns a detailed log of commits included in a PR check, starting with (and
- * including) the branch point off of master.
+ * including) the branch point off of master. Limited to at most 100 commits to
+ * keep the output sane.
  *
  * @return {string}
  */
 exports.gitDiffCommitLog = function() {
+  const maxCount = 100;
```

Comment:
Nit: rather have this at the top of the file

Reply:
Done.